src/main/_docs/setup/setup-bitnami.md

```diff
@@ -7,81 +7,4 @@ permalink: /:categories/bitnami/
 ---
```

Comment:
I'd like to leave the page there for awhile to avoid 404 errors from external sites. Bitnami will remove their Che cloud starters in the coming weeks but they still exist for now.